internal/binary/envoy/list.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoy
+
+import (
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/tetratelabs/getenvoy/internal/manifest"
+)
+
+// PrintVersions retrieves the manifest from the passed location and writes it to the passed writer
+func PrintVersions(m *manifest.Manifest, goos string, w io.Writer) {
+	p := manifest.BuildPlatform(goos)
+	// print nothing if the only released "flavor" or the platform doesn't exist
+	if m.Flavors["standard"] == nil || p == "" {
```

Comment:
Let's use `f, ok := m.Flavors["standard"]` here and the below for indexing with "standard".

Reply:
this code will die, but addressed this since you took time to suggest it!